package.json

```
@@ -61,6 +61,7 @@
         "lodash-es": "4.17.21",
         "markdown-it": "14.1.0",
         "markdown-it-class": "1.0.0",
+        "markdown-it-github-alerts": "^0.3.0",
```

```suggestion
        "markdown-it-github-alerts": "0.3.0",
```
All other dependencies use exact version specifiers.

---

Thanks for pointing that out @b-fein.
Totally missed that.